### ISSUE TITLE
Latest build is not populating config values

### ISSUE THREAD

**Issue by pajtai** [dylang/grunt-prompt#40]
Great grunt task, been using it for quite a while.

v1.3.2

sample:

```javascript
prompt : {
            deploy : {
                options : {
                    questions : [
                        {
                            config : 'environment',
                            type : 'list',
                            message : 'Environment to deploy to:',
                            default : 'staging',
                            choices : ['staging', 'production']
                        }
                    ]
                }
            }
        }
```

using the following will log and empty string here:

```javascript
    grunt.registerTask('logit', function() {
        console.log(grunt.template.process('the env is <%= environment %>'));
    });
    grunt.registerTask('deploy', [
        'prompt:deploy',
        'logit']);
```

> the env is

1.3.0 logs staging

**Comment by nvaken** [comment]
Same problem here! Adding fixed version v1.3.0 to our grunt projects for now until this issue is resolved.

**Comment by ld0rman** [comment]
Also experiencing this problem

**Comment by dylang** [comment]
Sorry about that! 

Thanks for quickly reporting it @pajtai and followups from @nvaken and @ld0rman.

It was a quick fix and I've published the new version as `1.3.3`.

**Comment by nvaken** [comment]
Awesome, thanks for the quick hotfix.
